### PR TITLE
fix(css reset/components): added important for slotted(*) targeted elements font styles

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Our CSS-reset has a rule:
```css
input,
button,
textarea,
select {
  font: inherit;
}
```
Which was causing issues for the Tabs. There selector (slotted(*)) for targeting slotted buttons was not strong enough and was getting overwritten by the rule above. This resulted in this look:
Screenshot 2023-09-28 at 13 55 39

When it should look like this:
Screenshot 2023-09-28 at 13 56 50

To combat this. I have used `!important` for the `font` and `letter-spacing` in our CSS whenever we are targeting `::slotted(*)`

**Solving issue**  
Fixes: [CDEP-2708](https://tegel.atlassian.net/browse/CDEP-2708)

**How to test**  

To see the error: 
1. Checkout the `develop` branch.
2. In `preview.css` add our CSS-reset
3. Check in Tabs and verify that the text have the wrong styling.

1. Checkout this branch.
3. In `preview.css` add our CSS-reset
5. Check in Tabs that the look of the tabs are intact.

[CDEP-2708]: https://tegel.atlassian.net/browse/CDEP-2708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alexandereneroth — APPROVED** [review]

File: packages/core/src/components/card/card.scss [hunk 1]
```diff
@@ -77,8 +77,8 @@
   }
 
   slot[name='body']::slotted(*) {
-    font: var(--tds-detail-03);
-    letter-spacing: var(--tds-detail-03-ls);
+    font: var(--tds-detail-03) !important;
```
Comment (timrombergjakobsson) [inline packages/core/src/components/card/card.scss:80]: I feel that using `!important` is a bit scary. But if it is the only way to fix this, then maybe have a comment about why doing it like this? Or potentially, but not part of this pr, look into the css reset we have and see if it can be replaced with something else that does not require the usage of `!important`

Comment (mJarsater) [reply packages/core/src/components/card/card.scss:80]: Agree, the `!important` seems to be necessary due to the weakness of the `::slotted` pseudo selector. But I will add comments like you suggested 🚀

**Review by timrombergjakobsson — APPROVED** [review]
